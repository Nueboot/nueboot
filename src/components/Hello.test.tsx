import { shallow, ShallowWrapper } from "enzyme";
import React from "react";

import Hello from "./Hello";

describe("Hello", () => {
  let rendered: ShallowWrapper;

  beforeEach(() => {
    rendered = shallow(<Hello name="mickey" />);
  });

  it("renders as expected", () => {
    expect(rendered).toMatchSnapshot();
  });
});
