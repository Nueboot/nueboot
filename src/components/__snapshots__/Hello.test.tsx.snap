// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello renders as expected 1`] = `
<h1>
  Hello 
  mickey
</h1>
`;
